examples/chip-tool/commands/clusters/ReportCommand.h

```diff
@@ -355,6 +355,7 @@ class SubscribeEvent : public SubscribeCommand
         AddArgument("fabric-filtered", 0, 1, &mFabricFiltered);
         AddArgument("event-min", 0, UINT64_MAX, &mEventNumber);
         AddArgument("keepSubscriptions", 0, 1, &mKeepSubscriptions);
+        AddArgument("isUrgent", 0, 1, &mIsUrgents, "Sets isUrgent in the Subscribe Request");
```

Comment:
Shall we include a condensed version of this description for `isUrgent` in the arg dec here? 

`The queueing of any urgent event SHALL force an immediate generation of reports containing all events queued leading up to (and including) the urgent event in question.`